subcommand/inject-connect/command.go

```diff
@@ -113,10 +126,22 @@ func (c *Command) init() {
 			"discovery across Consul namespaces. Only necessary if ACLs are enabled.")
 
 	// Resource setting flags.
```

Comment:
```suggestion
	// Sidecar resource setting flags.
```

Reply:
We have two sidecars so this isn't deterministic. Updated to `Proxy sidecar resource setting flags.`